flexport/latitude
# Fix issue where FilterButton errors on unmount in some cases.

## Description:
Fixes set state bug post unmount for `FilterButton`. Fixes an error that may be causing integration test to fail when upgrading Latitude in monorepo.

## Manual Test Plan:
This PR does not change anything, only fixes a React error. The behavior from this PR should be retained (https://github.com/flexport/latitude/pull/118).

1. Open Transmission App from `master`.
2. Visit `transmissionapp.com/loads/driver_load_updates`.
3. Observe the below error.
<img width="917" alt="Screen Shot 2019-12-18 at 6 00 13 PM" src="https://user-images.githubusercontent.com/7979583/71141535-46e86580-21c9-11ea-9ef6-373ed53646b6.png">

4. Update `yarn.lock` to this commit.
5. Observe the error no longer shows
